apps/admin_api/lib/admin_api/v1/router.ex

```diff
@@ -33,6 +38,12 @@ defmodule AdminAPI.V1.Router do
     post("/transaction.all", TransactionController, :all)
     post("/transaction.get", TransactionController, :get)
 
+    scope "/" do
+      pipe_through([:idempotency])
+
+      post("/transaction.create", TransactionController, :create)
```

Comment:
This is the same logic as eWallet API's `/transfer` right? Thinking if we should have consistent naming...

Reply:
Let's rename it in the eWallet API then.